### DISCUSSION TITLE
gestione bollo in fattura

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#245] [Q&A] [answered]
Al caricamento di una nuova fattura dove il bollo è obbligatorio come si gestisce che sia a carico o meno del cliente?
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @zpj61, in [questa discussion](https://github.com/fattureincloud/api/discussions/165) sono indicate le due modalità di inserimento del bollo in fattura.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Ciao @zpj61 
è riuscito a risolvere questo problema?

Grazie
Mauro
